### PR TITLE
chore: add report bug button

### PR DESCRIPTION
**Pull request by tmspzz**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

The application doesn't have a way for the user to easily report a bug

### Solutions

Add a menu item in the app's drawer to report a bug

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1041/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4525835)(4525835)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4525835)(4525835)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt [hunk 1]
```diff
@@ -168,18 +174,29 @@ fun LoggingSection(
     TextRowItem(
         "Share Logs",
         trailingIcon = android.R.drawable.ic_menu_share
-    ) { context.startMultipleFileSharingIntent(logFilePath()) }
+    ) {
+        val dir = File(logFilePath()).parentFile
+        val fileUris = context.getUrisOfFilesInDirectory(dir)
+        val intent = context.multipleFileSharingIntent(fileUris)
+        // The first log file is simply text, not compressed. Get its mime type separately
+        // and set it as the mime type for the intent.
```
Comment (MohammedMokresh) [inline app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt:182]: sorry here I didn't understand what we are trying to achieve, is it to make the shared files get the correct mimetype ?
if that's the case the original startMultipleFileSharingIntent function was doing that already. but in this case so we can just update that function to make it return intent so we can use it in the case of general share or the email share only below.

**Review by MohammedMokresh — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt (hunk 1 above)
Comment (tmspzz) [reply app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt:182]: Replied privately

File: app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt [hunk 2]
```diff
@@ -111,6 +120,22 @@ fun HomeDrawer(
...
+
+                val dir = File(viewModel.logFilePath()).parentFile
+                val logsUris = context.getUrisOfFilesInDirectory(dir)
+                val intent = context.multipleFileSharingIntent(logsUris)
+                intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
+                intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
+                intent.putExtra(Intent.EXTRA_TEXT, viewModel.reportBugEmailTemplate(context.getDeviceId()?.sha256()))
+                intent.type = "message/rfc822"
+
+                context.startActivity(Intent.createChooser(intent,"Choose an Email client : "))
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt:136]: Maybe we can extract this to a string resource as well ?  `Choose an Email client` 🤔

**Review by yamilmedina — APPROVED** [review]
I just tested it, looks good 👏  just a small suggestion if it makes sense at this point.

File: app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt (hunk 2 above)
Comment (tmspzz) [reply app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt:136]: when we localise it then yes. I am not sure this poor man's reporting button will stay after beta

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1041/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4530042)(4530042)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4530042)(4530042)-dev-release.apk)

**Pull request merged**